glastopf/glastopf.cfg.dist

```diff
@@ -105,3 +105,12 @@ sensorid = None
 
 [profiler]
 enabled = False
+
+[s3storage]
+enabled = False
+endpoint = http://localhost:8080/
+aws_access_key_id = YOUR_aws_access_key_id
+aws_secret_access_key = YOUR_aws_access_key_id
+bucket = glastopf
+region = europe
```

Comment:
Maybe use `eu-west-1`?

Reply:
This was meant as a dummy and needs to be set according to the configuration anyway. However, I set it to eu-west-1.  Changed in e906972037d6aed064d23b46a7a654ae62f6a1dc